notebooks/RAG_zephyr_langchain.ipynb

```diff
@@ -0,0 +1,11589 @@
+{
```

Comment:
<p>Same as above, maybe the output is not informative enough so just clear it?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>